TGC 과제 PR - Google 페이지 제작

HY0118 opened the pull request:
처음에 커밋 했다가 제 개인 저장소가 아니라 상위 저장소에 커밋한줄 알고 Revert를 했는데,,
사실 잘 커밋했었다는 것을 뒤늦게 깨닫고 revert의 revert를 해버려서 더미 커밋이 몇개 생겼습니다 ㅎㅎ

요약하자면,,
revert 커밋은 무시하시고 아래 두 커밋만 봐주시면 됩니다.

2494a3e08f5873802b05a02164ad1583c08224c6
5277bef83e308fca4200e55427278531ac7b47df

HY0118 committed:
TGC 과제 - google 첫 페이지 UI 제작

google favicon 파일 추가
html - body에 <div>컴포넌트로 영역 분리 및 기능 별 <input>, <a> 태그 적용
css - UI CSS 에 맞게 id로 설정 및 공용 CSS는 class로 분리

HY0118 committed:
각 버튼 동작 기능 추가

search 버튼 : input값을 받아와서 google에 검색하도록 구현
lucky 버튼 : 벚꽃 이미지 링크로 리다이렉트
Gmail nav : google gmail 페이지로 이동
Image nav : google image 검색 페이지로 이동

+ Enter 입력 시 Key 이벤트를 input에 붙여서 search 버튼과 동일한 동작을 하도록 구현

HY0118 committed:
Revert "각 버튼 동작 기능 추가"

This reverts commit 5277bef83e308fca4200e55427278531ac7b47df.

HY0118 committed:
Revert "TGC 과제 - google 첫 페이지 UI 제작"

This reverts commit 2494a3e08f5873802b05a02164ad1583c08224c6.

HY0118 committed:
Revert "Revert "TGC 과제 - google 첫 페이지 UI 제작""

This reverts commit 420debf6e4a1908c5463c4ef98f343b4ce6e0e53.

HY0118 committed:
Revert "Revert "각 버튼 동작 기능 추가""

This reverts commit 6c27bcb017212e9c39e056f6d3c3acfce65bc8d5.

creativeYS commented on practice01/google.css:
```diff
@@ -1,10 +1,125 @@
+	&:focus {
+		border: transparent;
+		box-shadow: 0 1px 6px rgba(32,33,36,.28);
+	}
```
오.. ㅋㅋㅋ 포커스 효과까지 적용하다니 역쉬..👍

creativeYS commented on practice01/google.html:
```diff
@@ -2,8 +2,37 @@
+	 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<title>Google</title>
+	<!-- favicon -->
+	<link rel="icon" href="./favicon.ico">
```
favcon... ㅋㅋㅋ 디테일도 좋아용

creativeYS commented on practice01/google.html:
```diff
@@ -2,8 +2,37 @@
 <body>
+	<div id="container">
+		<div id="navbar">
+			<div id="navgmail" class="nav_link_text" onClick={onClickGmail();}>
```
html에서 onclick 을 정의해주셔도 좋지만, html의 뼈대 개념과 js 코드의 분리
그리고 html 문서의 입장에서 연결된 문서를 알려준다는 점에서
단순 링크의 경우는 a 태그를 사용하는 것이 좋다고합니다 참고해주세요
`<a href="http://naver.com">link</a>`

creativeYS commented on practice01/google.html:
```diff
@@ -2,8 +2,37 @@
+		<div id="link_container">
+			<div id="links">
+				<div id="link_search" class="link_button_common" onClick={onClickSearch();}>
+					<a id="link_a_tag_text">Google Search</a>
```
input 에서 enter 키 입력 이벤트를 js에서 정의해준 것처럼 onclick 이벤트도 js에서 연결해주는 것도 좋을 것 같네요~

creativeYS commented on practice01/google.js:
```diff
@@ -1,5 +1,26 @@
+}
+
+function onClickLucky() {
+	window.location.href = 'https://images.pexels.com/photos/1043458/pexels-photo-1043458.jpeg';
```
ㅋㅋㅋㅋㅋㅋㅋㅋㅋ

creativeYS commented:
아주 깔끔하게 잘 작성해주셔서 감사합니다. ㅎㅎㅎ👍
소소한 피드백을 남겼으니 확인해보시고, 사실 병합해도 문제는 없을 거 같지만, 한번 피드백 및 수정후 병합하는 과정을 겪어보자고요~

HY0118 committed:
html과 js의 역할에 따라 코드 수정(이벤트 분리 및 태그 수정)

HTMl에서 onClick이벤트 제거 후, js에서 ID 기반으로 click 이벤트 추가
div 태그에 onClick이벤트로 새로운 페이지 이동하는 로직을 a 태그로 대체.

HY0118 committed:
a 태그 텍스트에 밑줄 그어지는 효과 제거

HY0118 commented:
@creativeYS 피드백 주신 부분 수정해서 커밋했습니다 👍

creativeYS commented:
고생많으셨습니다!!
병합하겠습니다.
